### ISSUE TITLE
Latest build broken

### ISSUE THREAD

**Issue by rhaamo** [VGoshev/seafile-docker#12]
Issue seems here: https://github.com/VGoshev/seafile-docker/blob/946714d85beb7a1137b5e51424b839455c7d66f8/docker/docker-run-root#L10

I think it's needed to add a space after ```[```

Log:
```
groupmod: PAM: Permission denied
usermod: no changes
uid=2016(seafile) gid=2016(seafile) groups=2016(seafile),2016(seafile)
/bin/docker-run-root: line 18: [seafile: not found
su-exec: setgroups: Operation not permitted
```

**Comment by VGoshev** [comment]
Ou, really, thanks. Although other errors:
`groupmod: PAM: Permission denied`
and
`su-exec: setgroups: Operation not permitted`
aren't related to this space issue. Do you have SELinux activated on your host?

**Comment by rhaamo** [comment]
no I don't have selinux at all.
does this use something specific to some docker version ?

**Comment by VGoshev** [comment]
Nope, nothing specific to docker, just switching from root to seafile user.
I'll think, what could cause this issue and how to fix it

**Comment by rhaamo** [comment]
I've only found https://github.com/ncopa/su-exec/issues/2 which talk about filesystem permissions.
